hoomd/hpmc/IntegratorHPMC.h

```diff
@@ -446,6 +446,8 @@ class PYBIND11_EXPORT IntegratorHPMC : public Integrator
         return m_patch;
         }
 
+    std::shared_ptr<PatchEnergy> m_patch; //!< Patchy Interaction
+
```

Comment:
? This shouldn't be public. It should be protected. Use `getPatchEnergy` to get it.

Reply:
Yes.